app/controllers/admin/configs_controller.rb

```diff
@@ -181,20 +41,11 @@ def confirmation_text
     end
 
     def config_params
-      all_params = ALLOWED_PARAMS |
-        CAMPAIGN_PARAMS |
-        COMMUNITY_PARAMS |
-        NEWSLETTER_PARAMS |
-        RATE_LIMIT_PARAMS |
-        MASCOT_PARAMS |
-        IMAGE_PARAMS |
-        ONBOARDING_PARAMS |
-        JOB_PARAMS
 
       has_emails = params.dig(:site_config, :email_addresses).present?
       params[:site_config][:email_addresses][:default] = ApplicationConfig["DEFAULT_EMAIL"] if has_emails
       params&.require(:site_config)&.permit(
-        all_params,
+        SiteConfig.keys.map(&:to_sym),
```

Comment:
I think the nature of this object is that it should be entirely readable and writeable by admins and no attribute is off limits per se, so I did away with our allowed list. It seems okay for this particular object.